strawberryfields/api/result.py

```diff
@@ -32,6 +32,9 @@ class Result:
 
     * ``results.samples``: Measurement samples from any measurements performed.
 
+    * ``results.ancilla_samples``: Measurement samples from any ancillary states
+        used for measurement-based gates.
```

Comment:
Worth double-checking the rendered version, this should adjust it.
```suggestion
    * ``results.ancilla_samples``: Measurement samples from any ancillary states
      used for measurement-based gates.
```